test/unittest/Database/DatabaseDumpTests.cpp

```diff
@@ -431,12 +439,6 @@ TEST(database, dump_unlinked_database)
     dump[PARTICIPANT_CONTAINER_TAG].begin().value()[PROCESS_ENTITY_TAG] = "-1";
     dump[PROCESS_CONTAINER_TAG].begin().value()[PARTICIPANT_CONTAINER_TAG] = DatabaseDump::array();
 
-    dump[DATAWRITER_CONTAINER_TAG].begin().value()[LOCATOR_CONTAINER_TAG] = DatabaseDump::array();
-    dump[DATAREADER_CONTAINER_TAG].begin().value()[LOCATOR_CONTAINER_TAG] = DatabaseDump::array();
-
-    dump[LOCATOR_CONTAINER_TAG].begin().value()[DATAWRITER_CONTAINER_TAG] = DatabaseDump::array();
-    dump[LOCATOR_CONTAINER_TAG].begin().value()[DATAREADER_CONTAINER_TAG] = DatabaseDump::array();
```

Comment:
Maybe we should add a test where no endpoint has been added yet or where some locator is not yet linked to an endpoint.